src/test/java/EntityExportDestinationSaveRecordTest.java

```diff
@@ -0,0 +1,61 @@
+import base.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static utils.ProjectUtils.start;
+import static utils.TestUtils.scrollClick;
+
+public class EntityExportDestinationSaveRecordTest extends BaseTest {
+
+    @Test
+
+    public void testSaveRecord() {
+
+        List<String> expectedValues = Arrays.asList(
+                "Some string", "Export destination text.", "457", "27.35",
+                "01/06/2021", "01/06/2021 13:07:06", "", "apptester1@tester.test");
+
+        start(getDriver());
+
+        scrollClick(getDriver(), getDriver().findElement(By.xpath("//p[contains(text(), ' Export destination ')]")));
+
+        getWait().until(ExpectedConditions.elementToBeClickable(getDriver().findElement(
+                By.xpath("//i[contains(text(), 'create_new_folder')]")))).click();
+
+        getDriver().findElement(By.id("string")).sendKeys("Some string");
+        getDriver().findElement(By.id("text")).sendKeys("Export destination text.");
+        getDriver().findElement(By.id("int")).sendKeys("457");
+        getDriver().findElement(By.id("decimal")).sendKeys("27.35");
+        getDriver().findElement(By.id("date")).click();
+        getDriver().findElement(By.id("date")).clear();
+        getDriver().findElement(By.id("date")).sendKeys("01/06/2021");
+        getDriver().findElement(By.id("datetime")).click();
+        getDriver().findElement(By.id("datetime")).clear();
+        getDriver().findElement(By.id("datetime")).sendKeys("01/06/2021 13:07:06");
```

Comment:
в BaseTest реализованы методы findElement() и findElements(), по этому getDriver() можно упустить
```suggestion
        findElement(By.id("string")).sendKeys("Some string");
        findElement(By.id("text")).sendKeys("Export destination text.");
        findElement(By.id("int")).sendKeys("457");
        findElement(By.id("decimal")).sendKeys("27.35");
        findElement(By.id("date")).click();
        findElement(By.id("date")).clear();
        findElement(By.id("date")).sendKeys("01/06/2021");
        findElement(By.id("datetime")).click();
        findElement(By.id("datetime")).clear();
        findElement(By.id("datetime")).sendKeys("01/06/2021 13:07:06");
```
и в других местах кода соответсвтенно

Reply:
Всё исправила, спасибо за замечания.